src/_content/posts/call-for-2025-volunteers.md

```diff
@@ -0,0 +1,319 @@
+- Once reviews are completed, de-anonymize talks and filter for duplicate speakers, multiple speakers from the same organization, diversity goals, overlapping topics, and more
+- Select the best talks (where "best" is extremely subjective; we have documentation on things we look for in good proposals); you're not strictly bound to pick the highest scoring talks, so you have some leeway
+- Coordinate with the Opportunity Grants team to send them a list of accepted speakers so they can prioritize them for awards
+- Coordinate with the OG team to notify speakers in waves (ideally completed by July 1)
+- Build a schedule (ideally by July 15)
+- Publish blog posts announcing speakers and keynotes (ideally by August 15)
+- Update schedules based on speakers' needs
+- Coordinate with DEFNA board to approve reimbursement requests
+- Generate cue sheets for session chairs and speaker mentors
+- Review online talks for Code of Conduct compliance
+- Recruit backup speakers in case of speakers having to back out (visa issues, illness, etc.)
+- Work with social events team to arrange speaker appreciation dinner
+- Generate spreadsheets in multiple formats to make other teams' lives easier
+
+### At-conference responsibilities
+
+- Work with registration desk to make sure that all speakers show up
+- Work with session managers (other organizers) to ensure distribution of cue sheets and speaker gifts
+- Check with tutorial speakers to make sure they have everything they need
+
+## Opportunity Grants Committee
+
+Size: 2 (plus reviewers)
+
+The OG committee is responsible for obtaining grants from the DSF and PSF and awarding grants to people who will best use them, prioritizing speakers, first time attendees, and people from underrepresented and/or historically disadvantaged groups.
+
+### Pre-conference responsibilities
+
+- Start call for grants process (concurrent with CFP)
+- Publish blog posts announcing grants process
+- Request grants from Django Software Foundation and Python Software Foundation (the DEFNA board will help you fill out the forms)
+- Recruit grant reviewers (other organizers and DEFNA members are happy to help)
+- Review submissions
+- Coordinate with program team about grants to speakers
+- Notify grant recipients in waves (concurrent with program notifications)
+- Communicate with recipients on how to get reimbursed (we'll have templates)
+- Give DEFNA a lit of recipients and amounts
+- Approve reimbursements as requests come in
+- Coordinate with visa (community) team as needed to approve invitation letter requests
+- Write transparency report giving a high-level overview of how the grants were awarded
+
+### At-conference responsibilities
+
+- Answer questions from grant recipients and the DEFNA board
+
+## Community Team
+
+Size: 2-4
+
+### Pre-conference responsibilities
+
+- Schedule orientation
+- Arrange childcare and lactation room
+- Coordinate visa applications
+- Improve accessibility of the venue and conference as a whole
```

Comment:
When I was gathering info on my responsibilities, improving accessibility didn't seem to be a high priority. Do we want to leave that on the list of responsibilities? (To be clear, I think it would be great to do and I'm sure there's room for improvement.)

Potential responsibilities to add:
- Organize hackathon
- Organize community interviews

Reply:
Personally, I think the hackathon actually belongs under the Social team and the community interviews probably under the Communications team. That said, how it worked out this year was fantastic. It was a good example of a person seeing something that was of interest of them and pushing it forward.